hugo/layouts/docs/single.html

```diff
@@ -8,11 +8,10 @@
     <div class="docs" data-docs>
         <div class="docs__main">
             {{ partial "paging/breadcrumb.html" . }}
-            {{ partial "separator.html" }}
 
             <article class="article article--docs">
                 <div class="article__container">
-                    {{ partial "article--header" . }}
+                    {{ partial "article--header" (dict "showTitle" false "context" .) }}
```

Comment:
with the design with breadcrumb above the title: shouldn't the title always show as the article_title?